float32[5]  MotorPos
float32[5]  MotorVel
float32[5]  MotorTorq
bool        ClawMode
